docs/topics/high-performance-computing.md

```diff
@@ -273,6 +275,21 @@ CycleCloud allows you to:
 - Use your current job scheduler and applications without modification
 - Take advantage of built-in autoscaling and battle-tested reference architectures for a wide range of HPC workloads and industries
 
+![CycleCloud Architecture Example](images/az-cyclecloud-arch01.png "CycleCloud Architecture Example")
+
+#### Comparison chart
```

Comment:
Any reason we have a comparison chart under (H4) Azure CycleCloud (H3)?  Should this be H3 content?